src/main/webapp/2Dscript.js

```diff
@@ -677,15 +677,17 @@ function computeSubSection(pose, currentLat, currentLng, priorLat, priorLng) {
   let discoveredMaxLngPair = -91;
   subData = [];
   for (point of pose) {
+    const loopLat = point.lat;
+    const loopLng = point.lng;
     if (withinBound(minLat, maxLat, minLng, maxLng, loopLat, loopLng)) {
```

Comment:
This was correct without the extra variables (I don't think they carry their weight). You can instead change from `loopLat` => `point.lat`, etc.